domain/domain.go

```diff
@@ -407,15 +407,17 @@ func (do *Domain) PrivilegeHandle() *privileges.Handle {
 	return do.privHandle
 }
 
-// StatsHandle returns the statistic handle.
-func (do *Domain) StatsHandle() *statistics.Handle {
+// GetOrCreateStatsHandle returns the statistic handle. If the stats handle doesn't exist, it will create a new one.
```

Comment:
Just a little question here. Is it a good convention to explicit to describe how we get it in the function name? Or simply it doesn't matter? For example, `GetStatisticsTableCache` will create a pseudo one if it does not exist, do we have to say it in the function name?